test_runner/src/main/kotlin/ftl/args/AndroidTestShard.kt

```diff
@@ -27,6 +27,23 @@ object AndroidTestShard {
             listOf(filteredTests.map(FlankTestMethod::testName))
     }
 
+    fun getAllLocalTestShardChunks(args: AndroidArgs): ShardChunks {
```

Comment:
`Local` means that this method doesn't accept test apks by gcs paths, right?

Reply:
Correct.